flocker/control/_clusterstate.py

```diff
@@ -21,8 +21,19 @@ class ClusterStateService(Service):
     :ivar DeploymentState _deployment_state: The current known cluster
         state.
     """
-    def __init__(self):
+    def __init__(self, reactor):
         self._deployment_state = DeploymentState()
+        # self._information_wipers = pmap() # map ``IClusterStateWipe.key`` to its instance.
+        # self._information_wipers_added = pmap() # map ``IClusterStateWipe.key`` to timestamp
+
+    def startService(self):
+        # Start LoopingCall that applies all wipers whose added time is
+        # more than ten seconds in the past.
```

Comment:
Maybe make this a MultiService with a child TimerService and leave start/stop glue up to MultiService